client/src/pages/Dashboard.js

```diff
@@ -107,9 +95,11 @@ function Dashboard() {
           subline="Marie Groß, Lucas Naas..."
         />
         <Card bgColor="secondaryDark" imgSrc={Addicon} alt="Plus Icon" />
+        <div></div>
```

Comment:
Why is this here?

Reply:
This is for fixing the overflow bug left and right of our Card-Container - since we couldn't just add a margin/padding.